### ISSUE TITLE
Register default.yml with set don't work

### ISSUE THREAD

**Issue by jenstornell** [getkirby/panel#851]
**In a plugin I do this:**

```php
$kirby->set('blueprint', 'default', $blueprintpath . DS . 'default.yml');
```

I also have deleted the `default.yml` from the original `blueprints` folder. It works with any other blueprint but not the default one.

>Panel Error:
Error without a useful message :(
It happened here:
File: /panel\app\src\panel\models\page\blueprint.php
Line: 85

**Comment by lukasbestle** [comment]
Does the file `$blueprintpath . DS . 'default.yml'` exist?

Could you please also put in a temporary debug statement or breakpoint in `panel\app\src\panel\models\page\blueprint.php` line 66 that outputs what `$file` is set to?

**Comment by jenstornell** [comment]
**Yes, my `$blueprintpath`is like this:**

```
$blueprintpath = __DIR__ . DS . 'panel' . DS . 'blueprints';
```

It works for all other blueprints, except `default.yml`.

**On line 66 I added:**

```
echo $file;
```

And it returns an empty string.

`$name` on line 65 returns `site`.

**Comment by lukasbestle** [comment]
An empty string or something like `false`? Could you please try `var_dump()`?

> $name on line 65 returns site.

That shouldn't be from the same run of the method. It should first try `site` and then `default`. The second run will throw the error.

**Comment by jenstornell** [comment]
```php
var_dump($file);
```

gives `null`.

**Comment by lukasbestle** [comment]
I can't reproduce this with Kirby 2.3.0 and the following setup:

1. Use the Kirby starterkit
2. Move `site/blueprints/default.yml` to `site/plugins/test/blueprints/default.yml`
3. Create a `site/plugins/test/test.php` with the following content:

```php
<?php

kirby()->set('blueprint', 'default', __DIR__ . DS . 'blueprints' . DS . 'default.yml');
```

The blueprint is loaded correctly for me.

**Comment by samnabi** [comment]
@lukasbestle I can reproduce this issue by following your steps, and then...

4. Delete all the `.yml` files within `site/blueprints`

**Comment by lukasbestle** [comment]
Thank you for the information. We will try to fix this in one of the next patch releases.

**Comment by lukasbestle** [comment]
It was one of those "order of execution" bugs. The Panel sets up the multilang setup before the plugins are loaded, so it doesn't respect registered blueprints.

I have included the fix in Kirby 2.3.2. Let's hope that it won't break plugins, we'll see.

**Comment by samnabi** [comment]
Hooray! Thank you team Kirby 👌🏽
